### ISSUE TITLE
E0211: Method has no argument (no-method-argument) in Python 3.8 classes with positional-only parameters

### ISSUE THREAD

**Issue by fsenart** [PyCQA/pylint#3161]
### Steps to reproduce

```py
# bug.py
class Foo:
    def __init__(self, bar, /):
        ...
```

```sh
pylint --disable C,R,W bug.py
```

### Current behavior

```txt
************* Module bug
bug.py:2:4: E0211: Method has no argument (no-method-argument)

--------------------------------------------------------------------
Your code has been rated at -6.67/10 (previous run: -6.67/10, +0.00)
```

### Expected behavior

```txt

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

### pylint --version output

```
pylint 2.4.2
astroid 2.3.1
Python 3.8.0b4 (default, Oct  2 2019, 21:06:16) 
[GCC 8.3.0]
```

**Comment by PCManticore** [comment]
Thanks for reporting the issue.
